app/script/auth/configureStore.js

```diff
@@ -45,9 +45,20 @@ export const configureStore = (thunkArguments = {}) => {
 
 const createMiddleware = thunkArguments => {
   const middlewares = [thunk.withExtraArgument(thunkArguments)];
-  window.localStorage.removeItem('debug');
+
+  let localStorage;
+  try {
+    localStorage = window.localStorage;
+  } catch (error) {}
```

Comment:
One cool thing that can be done with localStorage is to fallback to a in-memory localStorage (like https://www.npmjs.com/package/localstorage-memory).
So in the `catch` fallback, you can actually do `localStorage = inMemoryLocalStorage` and you don't need those `if (localStorage)` afterward.

But that might not be the scope of this PR at all, I am not sure